nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -817,4 +823,25 @@ Costmap2DROS::dynamicParametersCallback(std::vector<rclcpp::Parameter> parameter
   return result;
 }
 
+void Costmap2DROS::getCostCallback(
```

Comment:
This needs test coverage

Reply:
I'm just checking the response of the service for now, will it be okay?